p2p/simulations/mocker.go

```diff
@@ -102,6 +102,11 @@ func startStop(net *Network, quit chan struct{}, nodeCount int) {
 func probabilistic(net *Network, quit chan struct{}, nodeCount int) {
 	nodes, err := connectNodesInRing(net, nodeCount)
 	if err != nil {
+		log.Error(fmt.Sprintf("%s", err))
```

Comment:
This should be more informative, for example:

```
log.Error("error connecting nodes in a ring", "err", err)
```

Reply:
Actually the line is not even needed, as `connectNodesInRing` already logs error events before returning an `error`